test/universal.js

```diff
@@ -449,6 +449,17 @@ describe('Google Analytics :: Universal', function() {
       });
     });
 
+    describe('#refundedOrderPartial', function() {
+      it('should send the right data', function(done) {
+        var json = test.fixture('refunded-order-partial');
+        test
+          .set(settings)
+          .track(json.input)
+          .sendsAlmost(json.output, {ignored: ['qt']})
```

Comment:
because the test fixture never knows what the value of qt will be before the test runs. ignoring qt is a common practice in all of our GA tests.